conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -11,3 +11,6 @@
 
 # Add all the application routes to the prod.routes file
 ->         /                          prod.Routes
+
+DELETE         /api-definition/test-only/:serviceName/events                                  uk.gov.hmrc.apidefinition.controllers.APIDefinitionController.deleteEvents(serviceName: ServiceName)
```

Comment:
I think that the /test-only/ bit of the URL has to be at the beginning - see https://github.com/hmrc/third-party-application/blob/main/conf/testOnlyDoNotUseInAppConf.routes as an example

Reply:
I did think so as well, but looking at other apps in github it seemed that most put it after the application prefix. And the readme commect says tha NGINX blocks it as soon as it's included in the url. But I can change it if that's our convention

Reply:
Changed